source/ranges.tex

```diff
@@ -9341,6 +9357,550 @@
+\begin{itemdecl}
+constexpr @\exposid{iterator}@& operator-=(difference_type n)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+@\exposid{current_}@ -= n;
+@\exposid{pos_}@ -= n;
+return *this;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator==}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr bool operator==(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return x.\exposid{pos_} == y.\exposid{pos_};}
+\end{itemdescr}
+
+\indexlibrarymember{operator<=>}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr strong_ordering operator<=>(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return x.\exposid{pos_} <=> y.\exposid{pos_};}
+\end{itemdescr}
+
+\indexlibrarymember{operator+}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr @\exposid{iterator}@ operator+(const @\exposid{iterator}@& x, difference_type y)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto temp = x;
+temp += y;
+return temp;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator+}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+friend constexpr @\exposid{iterator}@ operator+(difference_type x, const @\exposid{iterator}@& y)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return y + x;}
+\end{itemdescr}
+
+\indexlibrarymember{operator-}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ operator-(const @\exposid{iterator}@& x, difference_type y)
+  requires @\libconcept{random_access_range}@<@\exposid{Base}@>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto temp = x;
+temp -= y;
+return temp;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator-}{enumerate_view::\exposid{iterator}}%
+\begin{itemdecl}
+constexpr difference_type operator-(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return x.\exposid{pos_} - y.\exposid{pos_};}
+\end{itemdescr}
+
+\rSec3[range.enumerate.sentinel]{Class template \tcode{enumerate_view::\exposid{sentinel}}}
+
+\indexlibraryglobal{enumerate_view::\exposid{sentinel}}%
+\begin{codeblock}
+namespace std::ranges {
+  template<@\libconcept{view}@ V>
+    requires @\exposconcept{range-with-movable-references}@<V>
+  template<bool Const>
+  class enumerate_view<V>::@\exposidnc{sentinel}@ {                           // \expos
```

Comment:
```suggestion
  class enumerate_view<V>::@\exposid{sentinel}@ {
```

Reply:
Ok, this does match how iterator was marked in [range.enumerate.iterator] ...

Reply:
Maybe it was an off-by-one error. See https://github.com/cplusplus/draft/pull/6135#discussion_r1113592850.